### ISSUE TITLE
Multiple senders sugar

### ISSUE THREAD

**Issue by emil14** [nevalang/neva#524]
Instead of

```neva
printer:sig -> (:num -> :sig)
fPrinter:sig -> (:num -> :sig)
```

Do

```neva
[printer:sig, fPrinter:sig] -> (:num -> :sig)
```

**Comment by emil14** [comment]
Can be implemented at parser level only:

1. Parse right side
2. Parse each sender
3. Create N connections for each sender

**Comment by emil14** [comment]
Implemented in #532
